specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/Microsoft.Storage/Storage.tsp

```diff
@@ -202,6 +226,14 @@ namespace Microsoft.EventGrid.SystemEvents {
     /** The type of blob. */
     blobType?: string;
 
+    /** The current tier of the blob. */
+    @madeRequired(ServiceApiVersions.v2024_01_01)
+    accessTier: StorageBlobAccessTier;
+
+    /** The previous tier of the blob. */
+    @madeRequired(ServiceApiVersions.v2024_01_01)
+    previousTier: StorageBlobAccessTier;
```

Comment:
Are these not exist before and been `@added` to v2024_01_01, or they are optional before and now been `madeRequired` in v2024_01_01 ?

Reply:
These did not exist before and are being added through this PR.

Reply:
Yeah, if this is new to 2024-01-01, it would be `@added`.

typespec guide https://typespec.io/docs/libraries/versioning/guide

If this is output-only model (user won't create this), adding "required" property does not count as cross api-version break.

Reply:
Removed the madeRequired decorator.